apertium-eng.eng.dix

```diff
@@ -12324,6 +12324,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="impunity">        <i>impunit</i><par n="bab/y__n"/></e>
 <e lm="impurity">        <i>impurit</i><par n="bab/y__n"/></e>
 <e lm="imputation">      <i>imputation</i><par n="house__n"/></e>
+<e lm="in-joke">      <i>in-jok</i><par n="house__n"/></e>
```

Comment:
"in-jok" --> "in-joke"?

Reply:
Isn't this how words with the "house" form are supposed to be written in the dictionary? That's what I understood from the other entries.

Reply:
My understanding was incorrect. Fixed